### PR TITLE
Cloud custom vm

### PR DESCRIPTION
**Pull request by felipemadero**
# Docs PR Template

## Why this should be merged

It describes how to deploy a custom vm intro cloud using CLI

## How this works

Gives the set of instructions to create the custom vm subnet, and then update the nodes

## How these changes were tested 

Manual execution of the given instructions

## To prevent any errors while building

- [] run `vale /docs/file/path` on all changed `.md` files to ensure all grammar rules pass
- [] run `markdownlint /docs/file/path` on all changed `.md` files to ensure all linting rules pass
- [] complete the above two checks and all additional rules outlined in `style-checker-notes.md` to
  ensure all checks pass

### If a document was removed/deleted

- [] the path to that doc must be redirected to a valid URL via the
  `_redirects` file
- [] `_redirects` were manually verified with the cloudflare preview link
- [] `sidebars.json` reflects all changes made to file path

### If a document was moved

- [] all files that were moved from their current directory to a new path have had their paths
  redirected via the `_redirects` file
- [] `_redirects` were manually verified with the cloudflare preview link
- [] `sidebars.json` reflects all changes made to file path

### CODE REVIEW

File: docs/tooling/cli-guides/upload-a-custom-vm-to-cloud.md [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+### Local Build Stage
+
+A locally compiled binary is needed to first set the CLI Subnet. You can create it by locally
+cloning HyperSDK, changing to the desired branch and then executing the build script. Eg:
+
+```bash
+git clone https://github.com/ava-labs/hypersdk
+cd hypersdk
+git checkout main
+./examples/tokenvm/scripts/build.sh <vmBinaryPath>
```
[the remark this answers is not in the document]
Comment (felipemadero) [reply docs/tooling/cli-guides/upload-a-custom-vm-to-cloud.md:59]: it is the path of the file. will add some description,

File: docs/tooling/cli-guides/upload-a-custom-vm-to-cloud.md [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+    "coldStorageValueReadUnits": 2,
+    "warmStorageKeyReadUnits": 1,
+    "warmStorageValueReadUnits": 1,
+    "storageKeyCreateUnits": 20,
+    "storageKeyValueUnits": 5,
+    "coldStorageKeyModificationUnits": 10,
+    "coldStorageValueModificationUnits": 3,
+    "warmStorageKeyModificationUnits": 5,
+    "warmStorageValueModificationUnits": 3,
+    "customAllocation": [
```
[the remark this answers is not in the document]
Comment (felipemadero) [reply docs/tooling/cli-guides/upload-a-custom-vm-to-cloud.md:117]: there is no such guide. will add a comment

File: docs/tooling/cli-guides/upload-a-custom-vm-to-cloud.md [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+    node-config.json
+    chain.json
+  ▸ subnet.json
+    per-node-chain.json
+```
+
+Provide the path to the Subnet config file:
+
+```text
+✗ Enter the path to your configuration file: <subnetConfPath>
```
[the remark this answers is not in the document]
Comment (felipemadero) [reply docs/tooling/cli-guides/upload-a-custom-vm-to-cloud.md:383]: I don't believe so. It is going to break the command interaction example.

**Review by meaghanfitzgerald — DISMISSED** [review]
LGTM. There was an issue where the entire page was encased in the `:::note` block on line 45 (I added the closure on line 47). Only really noticeable when run on local server or using the Cloudflare links attached to the PR, so definitely suggest using those links next time. 

As always, nice work and thanks for contributing to the docs!

**Review by meaghanfitzgerald — APPROVED** [review]

**Pull request merged**